src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -3718,6 +3769,10 @@ class StubGenerator: public StubCodeGenerator {
       MontgomeryMultiplyGenerator g(_masm, /*squaring*/true);
       StubRoutines::_montgomerySquare = g.generate_square();
     }
+
+    if (UseRVV) {
```

Comment:
I think we need another option to control this, maybe UseRVVForBigIntegerShiftIntrinsics. So that users can still control whether to enable this intrinsic when RVV is there.

Reply:
`UseRVVForBigIntegerShiftIntrinsics` option added. Thanks!